internal/librarian/configure.go

```diff
@@ -257,20 +257,24 @@ func shouldBeGenerated(serviceYamlPath, languageSettingsName string) (bool, erro
 	return false, nil
 }
 
-// Attempts to configure a single API. Steps taken:
+// configureAPI attempts to configure a single API. Steps taken:
```

Comment:
I think we *do* probably need to indent all these bullets (as you did in mergereleasepr.go for example). There may be others - I'm perfectly happy to chase this later if you want.